src/dynamicprompts/parser/parse.py

```diff
@@ -26,14 +37,15 @@
 
 
 class Parser:
-    def __init__(self, builder: ActionBuilder):
+    def __init__(self, builder: ActionBuilder, parser_config=default_parser_config):
```

Comment:
I don't think we should be adding new things into the already-deprecated class.

Reply:
Let's get rid of it then. I don't think we need to worry about backwards compatibility for any existing code